portal-api/src/v1/users/routes.js

```diff
@@ -156,25 +112,27 @@ module.exports.findById = (req, res, next) => {
   });
 };
 
-module.exports.updateById = (req, res, next) => {
+module.exports.updateById = async (req, res, next) => {
   try {
     const userIsAdmin = req.user?.roles?.includes(ADMIN);
     const userIsChangingTheirOwnPassword =
       req.user?._id?.toString() === req.params._id &&
       Object.keys(req.body).every((property) => ['password', 'passwordConfirm', 'currentPassword'].includes(property));
-      
+
```

Comment:
@abhi-markan this currently has some unintended behaviour -- covered in DTFS2-7031